e2etest/pkg/frr/bgp.go

```diff
@@ -32,6 +32,22 @@ func NeighborInfo(neighborName string, exec executor.Executor) (*Neighbor, error
 	return neighbor, nil
 }
 
+type NeighborsMap map[string]*Neighbor
+
+// NeighborsInfo returns informations for the all the neighbors in the given
+// executor.
+func NeighborsInfoMap(exec executor.Executor) (NeighborsMap, error) {
+	neighbors, err := NeighborsInfo(exec)
```

Comment:
just a thought, we could change this function to always return a map (but is not worth the refactoring)

Reply:
Let me check...

Reply:
done